salt/backups/unmount_drive.sls

```diff
@@ -0,0 +1,16 @@
+{% from "orchestrate/edx/backup" import backup with context %}
```

Comment:
This is trying to import a `backup` jinja variable from the orchestrate state which doesn't appear to be set. I'm also not in love with the idea of importing anything from the orchestrate state, I think that introduces too much tight coupling between them.
It should be possible to use something like ` {% set backup_volume_name = 'odl-operations-backups-cache-{}'.format(salt.grains.get('environment')) %}`